cime_config/testdefs/testmods_dirs/clm/FatesColdLUH2/README

```diff
@@ -0,0 +1,2 @@
+Currently the FATES LUH2 category of test mods currently only supports 
+4x5 grid resolutions.
```

Comment:
```suggestion
Currently the FATES LUH2 category of test mods currently only supports 
4x5 grid resolutions.  This is because we only have one LUH2 time series
dataset for the 4x5 resolution. In the future we will provide more resolutions
which will be added to the namelist defaults.
```